.github/workflows/apply_db_migrations_and_metadata.yml

```diff
@@ -0,0 +1,44 @@
+#
+# Applies database migrations to staging (master) and production
+#
+name: "Applies the migrations to the database"
+
+on:
+  push:
+    branches:
+      - main
+      - production
+    paths:
+      - "atd-vzd/**"
+      - ".github/workflows/apply_db_migrations_and_metadata.yml" 
+      - ".github/workflows/migration-helper.sh"
+
+jobs:
+  apply:
+    name: Apply Migrations
+    runs-on: ubuntu-20.04
```

Comment:
Do you think it would work to run this on 22.04? It's the newer LTS and will stay updated until '27, a couple of years longer than 20.04.

Reply:
yeah, let's go for it! Makes me think that we should look into updating this for other workflows running 20.04 too.